database/seeds/DatabaseSeeder.php

```diff
@@ -14,12 +14,17 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
         Eloquent::unguard();
-        DB::statement('SET FOREIGN_KEY_CHECKS = 0');
+
+        if (! app()->environment('testing')) {
+            DB::statement('SET FOREIGN_KEY_CHECKS = 0');
+        }
 
         $this->call('UsersSeeder');
         $this->call('TalksSeeder');
         $this->call('ConferencesSeeder');
 
-        DB::statement('SET FOREIGN_KEY_CHECKS = 1');
+        if (! app()->environment('testing')) {
+            DB::statement('SET FOREIGN_KEY_CHECKS = 1');
+        }
```

Comment:
Ah SQLite foreign key support, makes sense. Maybe we can remove those lines entirely? I can't think of a reason we'd need to disable foreign key checks when seeding, and if there is, maybe there's a way to fix that from a totally different angle.